Formula/spatialite-gui.rb

```diff
@@ -24,43 +23,39 @@ class SpatialiteGui < Formula
   depends_on "pkg-config" => :build
   depends_on "freexl"
   depends_on "geos"
-  depends_on "libgaiagraphics"
+  depends_on "libpq"
+  depends_on "librasterlite2"
+  depends_on "librttopo"
   depends_on "libspatialite"
-  depends_on "proj@7"
+  depends_on "libtiff"
+  depends_on "libxlsxwriter"
+  depends_on "lz4"
+  depends_on "minizip"
+  depends_on "openjpeg"
+  depends_on "proj"
   depends_on "sqlite"
+  depends_on "virtualpg"
+  depends_on "webp"
   depends_on "wxwidgets@3.0"
+  depends_on "xz"
+  depends_on "zstd"
 
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/85fa66a9/spatialite-gui/1.7.1.patch"
-    sha256 "37f71f3cb2b0b9649eb85a51296187b0adf2972c5a1d3ee0daf3082e2c35025e"
-  end
+  uses_from_macos "curl"
+  uses_from_macos "libxml2"
```

Comment:
Not too important, but Homebrew's `libxml2` is getting linked rather than using macOS version. Probably pulled in via `libspatialite`.

Shouldn't be an issue as indirect dependency avoids broken linkage. In future, may want to check if `libspatialite` truly needs a newer copy of `libxml2` than what is provided by system.